cdn/dev/keyboard-search/search.js

```diff
@@ -145,7 +145,7 @@ function process_response(q, res) {
           "</div>"+
         "</div>");
 
-      $('.title a', k).text(kbd.name).attr('href', '/keyboards/'+kbd.id+embed_query_q);
+      $('.title a', k).text(kbd.name).attr('href', '/keyboards/'+kbd.id+(kbd.match.tag ? '?tag='+kbd.match.tag+embed_query_x : embed_query_q));
```

Comment:
embed_query_x starts with '&' and embed_query_q starts with '?'. This could be better but it's not worth rewriting for the sake of this!